components/dataentry/event-details.html

```diff
@@ -71,6 +71,17 @@
 </div>
 <!-- coordinates end -->
 
+<div class="row" ng-if="currentStage.enableUserAssignment">
+    <div class="col-sm-12">
+        {{'assigned_user' | translate}}
+        <d2-assign-users-input d2-model="currentEvent"
+                        d2-disabled="!currentStage.access.data.write"
```

Comment:
Check the eventEditable() function here?

Reply:
This has now been changed.